app/interactors/backdate/destroy_interactor.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Backdate::DestroyInteractor
+  include Interactor
+
+  delegate :params, :edition, :user, to: :context
+
+  def call
+    Edition.transaction do
+      find_and_lock_edition
+      update_edition
+      create_timeline_entry
```

Comment:
We should update the preview here. It looks like that is a step that should be done in https://github.com/alphagov/content-publisher/blob/9eb34fc4080532947c0d3c3ae6b9cae246cefba1/app/interactors/backdate/update_interactor.rb also

Reply:
Now calling the Preview Service to update the Publishing API 👍